Modules/_blake2/blake2module.c

```diff
@@ -12,62 +12,82 @@
 
 #include "impl/blake2.h"
 
-extern PyTypeObject PyBlake2_BLAKE2bType;
-extern PyTypeObject PyBlake2_BLAKE2sType;
-
+extern PyType_Spec blake2b_type_spec;
+extern PyType_Spec blake2s_type_spec;
 
 PyDoc_STRVAR(blake2mod__doc__,
 "_blake2b provides BLAKE2b for hashlib\n"
 );
 
+typedef struct {
+    PyTypeObject* blake2b_type;
+    PyTypeObject* blake2s_type;
+} Blake2State;
+
+static inline Blake2State* blake2_get_state(PyObject *module)
```

Comment:
PEP 7 coding style:

```suggestion
static inline Blake2State*
blake2_get_state(PyObject *module)
```

Reply:
@vstinner hopefully the [pep7 linter issue](https://github.com/giampaolo/psutil/issues/1351) is fixed so you don't have to spend time on this..